src/mutation-hoc.tsx

```diff
@@ -12,11 +12,11 @@ import {
   GraphQLBase,
 } from './hoc-utils';
 
-export function mutation<
+export function withMutation<
```

Comment:
FYI - This is a breaking change and relies on a semver bump.

Reply:
My bad, we use rollup and this wasn't included before.

Reply:
Yeah, this has never been exported by the actual react-apollo module. Also was never documented as public api.